src/issue-activity.ts

```diff
@@ -20,12 +19,19 @@ import {
   IssueParams,
   PullParams,
 } from "./start";
-import logger from "./helpers/logger";
+import { ContextPlugin } from "./types/plugin-input";
 
 export class IssueActivity {
-  readonly _configuration: DataCollectionConfiguration = configuration.dataCollection;
+  private readonly _context: ContextPlugin;
+  readonly _configuration: DataCollectionConfiguration;
 
-  constructor(private _issueParams: IssueParams) {}
+  constructor(
+    context: ContextPlugin,
+    private _issueParams: IssueParams
```

Comment:
Yes it declares and assigns the variable to the class (sorry forgot to answer your comment).

It's a TypeScript shorthand just to avoid rewriting the same code to assign
https://dev.to/satansdeer/typescript-constructor-shorthand-3ibd